### PR TITLE
Docs: Add the opening talk in the 1218 meetup.

### PR DESCRIPTION
**Pull request by ThreeMonth03**
According to the discussion in the sciwork discord, I add the opening talk agenda in 1218 meetup.

### CODE REVIEW

File: content/pages/meetup/2024/1218-nycu.rst [hunk 1]
```diff
@@ -26,7 +26,11 @@ There will be many different subjects discussed at the same time.
...
 +-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
-| 19:30 ~ 20:00   | free chat                | modmesh discussion and training:                      | mini motorway simulator development:         |
+| 19:30 ~ 20:00   | opening talk, sciwork and development:                                                                                          |
+|                 | Introduce the sciwork.                                                                                                          |
+|                 |                                                                                                                                 |
++-----------------+--------------------------+-------------------------------------------------------+----------------------------------------------+
+| 20:00 ~ 21:00   | free chat                | modmesh discussion and training:                      | mini motorway simulator development:         |
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/1218-nycu.rst:33]: Modify the timetable.

File: content/pages/meetup/2024/1218-nycu.rst [hunk 2]
```diff
@@ -50,6 +50,13 @@ There will be many different subjects discussed at the same time.
 Subjects
 ------------------
 
+opening talk - sciwork and development:
+++++++++++++++++++++++++++++++++++++++++++++
+
+Introduce the community that facilitate growth 
+through code development 
+for scientific, numerical, and engineering applications.
+
```
Comment (ThreeMonth03) [inline content/pages/meetup/2024/1218-nycu.rst:59]: Add the description of the opening talk.

**Review by ThreeMonth03 — COMMENTED** [review]
@yungyuc @Vincent-Lien Could you please review this pull request?

**Review by yungyuc — APPROVED** [review]
LGTM

**Pull request merged**